Actions/AddExistingApp/AddExistingApp.ps1

```diff
@@ -3,7 +3,7 @@ Param(
     [string] $actor,
     [Parameter(HelpMessage = "The GitHub token running the action", Mandatory = $false)]
     [string] $token,
-    [Parameter(HelpMessage = "Specifies the parent telemetry scope for the Telemetry signal", Mandatory = $false)]
+    [Parameter(HelpMessage = "Specifies the parent telemetry scope for the telemetry signal", Mandatory = $true)]
     [string] $parentTelemetryScopeJson = '{}',

```

Comment:
if it is mandatory - there shouldn't be a default.
I think you should leave those as mandatory false.